src/components/molecules/MonacoPlaceholder/MonacoPlaceholder.tsx

```diff
@@ -16,25 +35,33 @@ export const MonacoPlaceholder = () => {
     <S.Container>
       <div>
         <S.ImageContainer>
-          <S.Image src={MonacoPlaceholderImage} />
+          <S.Image src={MonacoPlaceholderImageNew} />
         </S.ImageContainer>
-        <S.Title>Need inspiration? Try any of these</S.Title>
+        <S.Title>
+          Select a resource on the left to edit
+          <br />
+          Want to go further? Try these shortcuts
+        </S.Title>
+        <S.Text>Everything about your Cluster live</S.Text>
+        <S.Text>
+          <S.InfoLink onClick={handleHelmCharts}>Preview Helm Charts</S.InfoLink>
+        </S.Text>
+        <S.Text>
+          <S.InfoLink onClick={handleKustomization}>Preview Kustomization</S.InfoLink>
+        </S.Text>
         <S.Text>
-          <b>Navigate and Validate</b>
-          <span> Resources</span>
+          <S.InfoLink onClick={handleGitOperations}>Git operations</S.InfoLink>
         </S.Text>
         <S.Text>
-          <b>Preview</b>
-          <span> Kustomizations and Helm Charts</span>
+          <S.InfoLink onClick={handleValidation}>Validate your resources</S.InfoLink>
         </S.Text>
         <S.Text>
-          <b>Compare</b>
-          <span> sets of Resources</span>
+          <S.InfoLink href="https://chrome.google.com/webstore/detail/monokle-cloud-chrome-exte/loojojkleiolidaodalflgbmaijeibob?hl=en">
```

Comment:
Here we need to use `shell.openExternal(link)`, so that the link is opened in your browser, not on the monokle app itself